src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -8,151 +8,134 @@ if [ ! -d "$PIP_CACHE_DIR" ];then
 fi
 
 {{ if VirtualEnvironmentName | IsNotBlank }}
+	{{ if PackagesDirectory | IsNotBlank }}
+		if [ -d "{{ PackagesDirectory }}" ]
+		then
+			rm -fr "{{ PackagesDirectory }}"
+		fi
+	{{ end }}
 
-{{ if PackagesDirectory | IsNotBlank }}
-if [ -d "{{ PackagesDirectory }}" ]
-then
-	rm -fr "{{ PackagesDirectory }}"
-fi
-{{ end }}
-
-VIRTUALENVIRONMENTNAME={{ VirtualEnvironmentName }}
-VIRTUALENVIRONMENTMODULE={{ VirtualEnvironmentModule }}
-VIRTUALENVIRONMENTOPTIONS={{ VirtualEnvironmentParameters }}
-zippedVirtualEnvFileName={{ CompressedVirtualEnvFileName }}
+	VIRTUALENVIRONMENTNAME={{ VirtualEnvironmentName }}
+	VIRTUALENVIRONMENTMODULE={{ VirtualEnvironmentModule }}
+	VIRTUALENVIRONMENTOPTIONS={{ VirtualEnvironmentParameters }}
+	zippedVirtualEnvFileName={{ CompressedVirtualEnvFileName }}
 
-echo "Python Virtual Environment: $VIRTUALENVIRONMENTNAME"
+	echo "Python Virtual Environment: $VIRTUALENVIRONMENTNAME"
 
-echo Creating virtual environment ...
-$python -m $VIRTUALENVIRONMENTMODULE $VIRTUALENVIRONMENTNAME $VIRTUALENVIRONMENTOPTIONS
+	echo Creating virtual environment...
+	$python -m $VIRTUALENVIRONMENTMODULE $VIRTUALENVIRONMENTNAME $VIRTUALENVIRONMENTOPTIONS
 
-echo Activating virtual environment ...
-source $VIRTUALENVIRONMENTNAME/bin/activate
+	echo Activating virtual environment...
+	source $VIRTUALENVIRONMENTNAME/bin/activate
 
-if [ -e "requirements.txt" ]
-then
-	echo
-	echo "Upgrading pip..."
-	START_TIME=$SECONDS
-	pip install --upgrade pip
-	ELAPSED_TIME=$(($SECONDS - $START_TIME))
-	echo "Done in $ELAPSED_TIME sec(s)."
-
-	echo "Running pip install..."
-	pip install --cache-dir $PIP_CACHE_DIR --prefer-binary -r requirements.txt | ts $TS_FMT
-	pipInstallExitCode=${PIPESTATUS[0]}
-	if [[ $pipInstallExitCode != 0 ]]
+	if [ -e "requirements.txt" ]
 	then
-		exit $pipInstallExitCode
-	fi
-elif [ -e "setup.py" ]
-then
-	echo
-	START_TIME=$SECONDS
-	pip install --upgrade pip
-	ELAPSED_TIME=$(($SECONDS - $START_TIME))
-	echo "Done in $ELAPSED_TIME sec(s)."
-	
-	echo "Running python setup.py install..."
-	$python setup.py install --user| ts $TS_FMT
```

Comment:
Changing from `$python` to `python` since we are executing this from within the virtual environment itself.